### PR TITLE
Db/docs edit mode

### PR DESCRIPTION
**Pull request by pawellysy**

### CODE REVIEW

File: docs/dashboards/edit-mode.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Edit mode
+===
+
+Edit mode is a mode, in which the user can change the appearance of the dashboard through the UI. To turn on this feature, you need to add the class to main dashboard div:
+```html
+<div id="container" class="highcharts-dashboard"></div>
```
Comment (karolkolodziej) [inline docs/dashboards/edit-mode.md:6]: Add `Tab`

File: docs/dashboards/edit-mode.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                items: ['editMode']
+            }
+        },
+```
+
+Then, the edit mode option can be enabled by the context menu in the upper right corner.
+
+![edit-mode-context-menu.png](edit-mode-context-menu.png)
+
+When edit mode is enabled, the appearance of the dashboard changes. When a cell is hovered it gets blue border and the row, in which the cell is, gets green border. Additionally both row and cell get a tooltip, wich allows the user to change its position, change its options, or just delete it.
```
Comment (karolkolodziej) [inline docs/dashboards/edit-mode.md:25]: Typo `wich`

File: docs/dashboards/edit-mode.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+```html
+<div id="container" class="highcharts-dashboard"></div>
+```
+
+
+Also in the dashboard config, this option needs to be enabled:
+```javascript
+    editMode: {
+            enabled: true,
+            contextMenu: {
```
Comment (karolkolodziej) [inline docs/dashboards/edit-mode.md:14]: We also should describe what can be configured in the `contextMenu`.

**Review by karolkolodziej — CHANGES_REQUESTED** [review]

File: docs/dashboards/edit-mode.md [hunk 4]
```diff
@@ -32,4 +32,18 @@ The “Large”, “medium”, and “small” buttons change the width of the d
 
 The `Add` button allows the user to add the new component. When clicked, the sidebar is shown, which lets you choose the type of component you want to add, and then by drag&drop component type can be selected and dragged to the correct place, which is also indicated by the drop marker.
 
-![edit-mode-sidebar.png](edit-mode-sidebar.png)
\ No newline at end of file
+![edit-mode-sidebar.png](edit-mode-sidebar.png)
+
+the `contextMenu` option also allows you to edit, what should be inside the menu, which shows after clicking on the burger menu.  The items can either be a string like `editMode` if it is a default button, or an object, which defines the button name, `onclick` event and some more options. Here is the example snippet of context menu button configuration:
```
Comment (karolkolodziej) [inline docs/dashboards/edit-mode.md:37]: Capital `T`

**Review by karolkolodziej — APPROVED** [review]

File: docs/dashboards/edit-mode.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+                items: ['editMode']
+            }
+        },
+```
+
+Then, the edit mode option can be enabled by the context menu in the upper right corner.
+
+![edit-mode-context-menu.png](edit-mode-context-menu.png)
+
+When edit mode is enabled, the appearance of the dashboard changes. When a cell is hovered it gets blue border and the row, in which the cell is, gets green border. Additionally both row and cell get a tooltip, which allows the user to change its position, change its options, or just delete it.
```
Comment (bre1470) [inline docs/dashboards/edit-mode.md:25]: `row, in which the cell is` => `row, containing the cell,` or at least `row, in which the cell is present`

**Review by sebastianbochan — APPROVED** [review]

**Pull request merged**